### DISCUSSION TITLE
Compile error : 'SocketIO' is a namespace but is used like a type

### DISCUSSION THREAD

**Discussion by imflammable** [doghappy/socket.io-client-csharp#382] [Q&A]
Getting this compile error when i update the SocketIOClient version to latest stable i.e 3.1.1

**Comment by tryingtofindanunusedusername** [comment]
I had the same problem. Try SocketIOClient.SocketIO

The example "Allowing Untrusted SSL Certificates" (_, _, _, _) doesn't work either. For me it works with (a, b, c, d) ... just for a try.

Unfortunately not all methods are named the same as in SocketIOClient.NetFx and the socket options attribute "EnabledSslProtocols" doesn't exist either.
